kolibri/plugins/facility_management/assets/src/views/class-edit-page/user-remove-modal.vue

```diff
@@ -6,8 +6,8 @@
     @cancel="close"
   >
     <div>
-      <span v-html="formattedDeleteConfirmation"> </span>
-      <p v-html="formattedAccessReassuranceConfirmation"> </p>
+      <p>{{ $tr('confirmation', { username: username, classname: classname }) }}</p>
+      <p>{{ $tr('description') }} <strong>{{ $tr('users') }}</strong>.</p>
```

Comment:
See [this comment](https://github.com/learningequality/kolibri/pull/3041#issuecomment-357313515) on why we can't use this kind of construction.

For now, we really need to remove _all_ formatting inside translated strings :)